courses/machine_learning/deepdive2/launching_into_ml/labs/first_model.ipynb

```diff
@@ -0,0 +1,461 @@
+{
```

Comment:
<p>Imports should be moved to a cell at the top.</p><p>Also, should project be replaced with "YOUR-PROJECT"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/654/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Done</p>